sycl/CMakeLists.txt

```
@@ -8,10 +8,10 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 option(SYCL_ENABLE_WERROR "Treat all warnings as errors in SYCL project" OFF)
 option(SYCL_ADD_DEV_VERSION_POSTFIX "Adds -V postfix to version string" ON)
 
-set(SYCL_MAJOR_VERSION 0)
+set(SYCL_MAJOR_VERSION 1)
```

@garimagu the major version is updated once per release. You only need to update DEV_ABI version.
```suggestion
set(SYCL_MAJOR_VERSION 0)
```

---

Ok.

---

uploaded the change. please review and approve. smaslov-intel has approved the changes other than the ABI version update.

---

I think we should (semi-)automate SYCL version update.
It's hard to manage multiple commits requiring version update.

@alexbatashev, could you create a post-commit job, which will create a PR with the version update if it's required, please?

---

@bader I'll take a look into it